### DISCUSSION TITLE
kyuubi.kubernetes.spark.cleanupTerminatedDriverPod.kind=ALL， Spark-SQL engine's driver was not cleaned up

### DISCUSSION THREAD

**Discussion by hippozjs** [apache/kyuubi#6809] [Q&A]
In the kyuubi-default.conf file, the configuration kyuubi.kubernetes.spark.cleanupTerminatedDriverPod.kind=ALL has been set. It was observed that on Kubernetes, the driver started via batch-submitted Spark JAR was cleaned up as expected after termination. However, the driver started by Spark SQL engine was not cleaned up after termination. 
kyuubi version is 1.8.1
Does it indicate a bug, or is additional configuration required?

**Comment by pan3793** [comment]
this is likely due to wrong configurations that cause the Kyuubi server fail to construct a Java K8s client

**Reply by hippozjs** [reply]
Which wrong configuration items might be problematic? Why is it that the batch jobs are cleaned up successfully。。。thanks
